contracts/src/core/AlignedLayerServiceManagerStorage.sol

```diff
@@ -11,6 +11,11 @@ abstract contract AlignedLayerServiceManagerStorage {
     // KEY is keccak256(batchMerkleRoot,senderAddress)
     mapping(bytes32 => BatchState) public batchesState;
 
+    // Bitmap representing blacklisted verifiers
+    // Each verifier is blacklisted if its corresponding bit is set to 1
+    // The verifier index follows its corresponding value in the `ProvingSystemId` enum being 0 the first verifier.
+    uint256 public blacklistedVerifiers;
```

Comment:
This variable should be added after `address public alignedAggregator;` because if not, it will shift the current storage and will break it.

Also, the GAP should be reduced, in this case, by 1, because we are adding an uint256